### DISCUSSION TITLE
Clarification on "management areas"

### DISCUSSION THREAD

**Discussion by zoyafuso-NOAA** [afsc-gap-products/StationAllocationAIGOA#15] [General]
@Ned-Laman-NOAA @MargaretSiple-NOAA 

I'll start with the question: **Historically, would you provide biomass estimates by INPFC areas AND by the NMFS statistical reporting areas?**

context:
A reminder that another change in the design is the use of management area as a stratum. In the historical design, the INPFC statistical areas are used as a stratum variable, consisting of five areas: Shumagin, Chirikof, Kodiak, Yakutat, and Southeastern are the names used in Oracle. The updated design uses instead the NMS regulatory areas as a stratum variable, consisting of three areas (with associated NMFS area codes), Western (610), Chirikof (620) and Kodiak (630), West Yakutat (640) and Southeast Outside (650). Chirikof and Kodiak are often merged as Central GOA and Yakutat and SE are merged as Eastern GOA. This was recommended by the Plan Team to be consistent with the areas used for apportionments. 

This might be confusing because I use the boundaries of the regulatory areas but use the INPFC names, and there are differences in the some of the boundaries of the two organizational naming conventions. Most drastic are the longitudinal boundary between Yakutat and SE and the INPFC Kodiak includes upper Shelikof.   

Another reason why I am asking this is because as I was in the process of preparing the new stations shapefile, I realized both INPFC_AREA and REGULATORY_AREA are fields in the GOA.GOAGRID_GIS. 

Thanks

**Comment by Ned-Laman-NOAA** [comment]
Answering your question:

HIstorically, we provided biomass (and lengthcomp) estimates by INPFC area AND by Depth Strata nested within INPFC areas AND we produced similarly structured tables pinned to the NMFS areas.

However, I understand that the boundaries of the NMFS and INPFC areas don't fully coincide and I **suspect**  that we were using INPFC boundaries to represent the NMFS areas in our stratum definitions. 

Do you have layers of both the NMFS and INPFC management zone boundaries to share? I'd like to see what I've only read about at this point.

**Reply by MargaretSiple-NOAA** [reply]
According to Sean, OFIS U drive has shapefiles for both the NMFS and INPFC areas. I will contact them today to see if I can get access but you and Zack may already have it.

**Reply by Ned-Laman-NOAA** [reply]
I can see the NMFS areas on OFIS' drive. 
The are @U:\alaska\constructs\zones\management\gf95_nmfs
I don't see anything here related to the INPFC bounds - I have some hand drawn INPFC boundaries in the Aleutians and I traced those from the aigrid shapefile so worse comes to worse could trace the INPFC bounds from the goagrid shapefile too

**Reply by MargaretSiple-NOAA** [reply]
Here are INPFC area shapefiles from Cheryl Barnes, who is the best: https://drive.google.com/drive/folders/1qqFs8gVdpy08Hd8tPcsKczHrxyPNXLLU?usp=sharing

**Reply by zoyafuso-NOAA** [reply]
I'll take a look a that, thanks Megsie.

For the boundaries, I think we just have to communicate to stock assessment that we are moving away from the INPFC boundaries and are moving towards consistency with the NMFS boundaries as defined in the GOA GF FMP (releated our email on that).

**Reply by Ned-Laman-NOAA** [reply]
I agree that our communication to the SAs should focus on how we're switching to NMFS areas and the advantages of that.
For what it's worth, the INPFC bounds from Cheryl and the NMFS boundaries line up pretty well; they are identical except for a few excursions in SEAK. What we've been calling INPFC bounds in the GOAGRID line up in a lot of places but not all. E.g., in SEAK NMFS area 659 is not a separate management zone in our historic survey stratification. Do we treat 659 separately from 650 in the restratification?

**Comment by Ned-Laman-NOAA** [comment]
I'm still trying to get this straight so I can let Sandra and Chris know what Oracle tables we'll be producing in the future. 

We presently produce BIOMASS_AREA, BIOMASS_AREA_DEPTH, SIZECOMP_AREA, and SIZECOMP_AREA_DEPTH tables organized by REGULATORY_AREA_NAME (WESTERN_GOA, CENTRAL_GOA, EASTERN_GOA). We will keep populating those tables.

We will no longer be producing the INPFC analogs of the above. 

However, I think we'll be producing tables BIOMASS_NMFS, BIOMASS_NMFS_DEPTH, SIZECOMP_NMFS, and SIZECOMP_NMFS_DEPTH tables organized by the 5 NMFS-defined management zones within the 3 REGULATORY_AREA_NAME areas above, right? And, those 5 NMFS-defined zones have similar names to the INPFC names but different boundaries than the INPFC boundaries, right?

If so, then these will be new tables not previously produced.

Please confirm if I have this right or straighten me out if I've got it wrong.

**Comment by zoyafuso-NOAA** [comment]
Yeah, and it may be good to clarify that the BIOMASS_AREA_DEPTH,
BIOMASS_NMFS_DEPTH, SIZECOMP_NMFS_DEPTH, and SIZECOMP_AREA_DEPTH tables
will have different depth zones in each area so it may be good to ask if
that is relevant going forward.

On Tue, Oct 18, 2022 at 7:12 AM Ned Laman ***@***.***> wrote:

> I'm still trying to get this straight so I can let Sandra and Chris know
> what Oracle tables we'll be producing in the future.
>
> We presently produce BIOMASS_AREA, BIOMASS_AREA_DEPTH, SIZECOMP_AREA, and
> SIZECOMP_AREA_DEPTH tables organized by REGULATORY_AREA_NAME (WESTERN_GOA,
> CENTRAL_GOA, EASTERN_GOA). We will keep populating those tables.
>
> We will no longer be producing the INPFC analogs of the above.
>
> However, I think we'll be producing tables BIOMASS_NMFS,
> BIOMASS_NMFS_DEPTH, SIZECOMP_NMFS, and SIZECOMP_NMFS_DEPTH tables organized
> by the 5 NMFS-defined management zones within the 3 REGULATORY_AREA_NAME
> areas above, right? And, those 5 NMFS-defined zones have similar names to
> the INPFC names but different boundaries than the INPFC boundaries, right?
>
> If so, then these will be new tables not previously produced.
>
> Please confirm if I have this right or straighten me out if I've got it
> wrong.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/afsc-gap-products/StationAllocationAIGOA/discussions/15#discussioncomment-3906633>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AOMCLOM7FD3UJFLAGICOHITWD2V3TANCNFSM6AAAAAARCWRT64>
> .
> You are receiving this because you authored the thread.Message ID:
> <afsc-gap-products/StationAllocationAIGOA/repo-discussions/15/comments/3906633
> @github.com>
>


-- 
*Zack Suriya Oyafuso, Ph.D.*
(he/him/his)
Research Fish Biologist
GitHub: github.com/zoyafuso-NOAA


NOAA Fisheries, Alaska Fisheries Science Center
Resource Assessment and Conservation Engineering Division
Groundfish Assessment Program, AI-GOA

7600 Sand Point Way NE, Bldg 4
Seattle, Washington 98115
Google Voice: (206) 526-4026

**Comment by Ned-Laman-NOAA** [comment]
OK. It sounds to me like the depth zones will be unique within the 5 NMFS areas and not within the larger 3 zones (also NMFS areas?) currently reflected in the *_AREA_DEPTH tables. To me it sounds like those *_AREA_DEPTH tables will now be obsolete. Is that your read?

I'm still wondering by how much do the boundaries and areas change within the 5 NMFS areas compared to the 5 INPFC areas. This isn't necessarily to communicate to the SAs, but mainly to satisfy my own curiosity. Cheryl's INPFC area boundaries differed very little from the NMFS areas which makes me suspicious.

**Reply by zoyafuso-NOAA** [reply]
That is correct, depth zones will be unique to each of the five areas. The *AREA_DEPTH tables will be obsolete with this stratification. 

I don't have the polygons of the INPFC areas used for the survey but since the historical strata are broken down by those divisions (the middle "tens" digit gives you the INPFC area), I have plotted below strata by INPFC area by color and then the NMFS areas superimposed. 

You asked about NMFS area 659 in the SE area earlier in this thread and yes, the historical goa grid and strata do creep into that area and as a result, our analysis and these new stations include some parts of NMFS area 659. 

That red bit sticking out in the Shumagin region north of the NMFS boundary is part of the division between historical strata 10 (Fox Islands) and 11 (Davidson Bank) on the West of Unimak Island. 

![strata_ontopof_area](https://user-images.githubusercontent.com/60302777/196558612-64b62da1-e7ff-4659-8d72-8ab77c3abf81.png)

**Reply by Ned-Laman-NOAA** [reply]
This is perfect, Zack. Thank you. I wanted to make sure I wasn't crazy when I was remembering differences between INPFC and NMFS areas. 

That's good to know about 659 and makes me realize that in our stratum lookup we will have to be able to separate stations in 650 from stations in 659. For example, if someone requests biomass for Stratum 650, we must be able to exclude CPUE in 659 and exclude the 659 hectares from the 650 area.
